### PR TITLE
APP-1538 : Fetch Multisig settings data

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

- Created separate plugins containers.
- Show settings details for multisig DAOs (example: 0x84432686c0d14f362e0e7c08c780682116d6bc44)

Task: [APP-1538](https://aragonassociation.atlassian.net/browse/APP-1538)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1538]: https://aragonassociation.atlassian.net/browse/APP-1538?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/settings/majorityVoting/index.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      <DescriptionListContainer title={t('navLinks.community')}>
+        <Dl>
+          <Dt>{t('labels.review.eligibleVoters')}</Dt>
+          <Dd>
+            {isErc20Plugin
+              ? t('createDAO.step3.tokenMembership')
+              : t('createDAO.step3.walletMemberShip')}
+          </Dd>
+        </Dl>
+        {isErc20Plugin && (
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/settings/majorityVoting/index.tsx]: So at this point, would this check not be redundant? Or are you expecting other token standards to be allowed in Majority Voting? But if so, wouldn't they be handled in a different component/file?

**Review by WissenIstNacht — APPROVED** [review]
Thanks man. Letf a question regardin code structure, but otherwise, good to go!

File: packages/web-app/src/containers/settings/majorityVoting/index.tsx (hunk 1 above)
Comment (Rekard0) [reply packages/web-app/src/containers/settings/majorityVoting/index.tsx]: true, amended

**Review by Fabricevladimir — APPROVED** [review]
Works for me

**Pull request merged**